src/tasks/component-installers/devfile-workspace-operator-installer.ts

```diff
@@ -273,21 +386,48 @@ export class DevWorkspaceTasks {
           if (await this.kubeHelper.clusterRoleBindingExist(this.devWorkspaceRoleBinding)) {
             await this.kubeHelper.deleteClusterRoleBinding(this.devWorkspaceRoleBinding)
           }
+          if (await this.kubeHelper.clusterRoleBindingExist(this.devworkspaceProxyClusterRoleBinding)) {
```

Comment:
You changed this task to remove two rolebindings instead of one. Remane task name please:
from
title: 'Delete ClusterRoleBinding for DevWorkspace controller',
to
title: 'Delete ClusterRoleBindings for DevWorkspace controller',